modules/openy_features/openy_menu/openy_menu.install

```diff
@@ -15,9 +15,17 @@ function openy_menu_update_8001() {
   $config .= 'tour.tour.openy-menu-overview.yml';
   $config_importer->update($config, 'tour.tour.openy-menu-overview', 'tips');
 }
+
 /**
  * Update menu tutorials.
  */
 function openy_menu_update_8002() {
   openy_menu_update_8001();
 }
+
+/**
+ * Update menu tutorials.
+ */
+function openy_menu_update_8003() {
+	openy_menu_update_8001();
```

Comment:
It's better to copy & paste code that triggering updates in updates. The same for update above.